distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestPlatform.java

```diff
@@ -413,4 +421,12 @@ public void testConfigurationFileWithInvalidFormat() throws IOException {
         invalidConfig.addConfigurationFile();
         invalidConfig.assertFileMovedToFailedDirectory();
     }
+
```

Comment:
@figliold created a new class for the platform commands. You may want to talk to him and see which way you guys want to do this so they're all in the same file. Also, we probably need at least one more test that does export something and checks that the files are actually exported to the export directory. We may also want one that test writing the files to a read-only or non-existing directory and make sure the right error message is being reported.